packages/nimble-components/src/table/testing/table.pageobject.ts

```diff
@@ -186,6 +188,34 @@ export class TablePageObject<T extends TableRecord> {
         return anchor as Anchor;
     }
 
+    public getRenderedCellIcon(
+        rowIndex: number,
+        columnIndex: number
+    ): Icon | null {
+        const icon = this.getRenderedCellView(rowIndex, columnIndex).shadowRoot!
+            .firstElementChild;
+        if (!icon || !(icon instanceof Icon)) {
+            throw new Error(
+                `Icon not found at cell ${rowIndex},${columnIndex}`
+            );
+        }
+        return icon;
+    }
+
+    public getRenderedCellSpinner(
```

Comment:
I was wondering the benefit of actually returning the `Spinner` instance here since I don't think there is much relevant state on it, so I did a code search to see how it's used. I don't actually see it used in any of the tests, so can it just be removed?